src/Recommendations.js

```diff
@@ -33,10 +33,19 @@ const getRecommendedObject_TEMPORARY_BETA = (
     });
 };
 
+const composeFilters = (recoFilters) => {
+  let compose = '';
+  if (recoFilters.length > 0) {
+    compose += `(${recoFilters.join(' OR ')})`;
+  }
+  return compose;
+};
```

Comment:
This is equivalent to:

```suggestion
const composeFilters = (recoFilters) => recoFilters.join(' OR ');
```

Reply:
yes thanks, I didn't clean after removing the negative filter from the recipe!